crates/bevy_input/src/gamepad/system.rs

```diff
@@ -0,0 +1,120 @@
+use crate::{
+    gamepad::{
+        GamepadAxis, GamepadButton, GamepadEvent, GamepadEventRaw, GamepadEventType,
+        GamepadSettings, Gamepads, ALL_AXIS_TYPES, ALL_BUTTON_TYPES,
+    },
+    Axis, Input,
+};
+use bevy_app::{EventReader, EventWriter};
+use bevy_ecs::system::{Res, ResMut};
+use bevy_utils::tracing::info;
+
+/// Monitors gamepad connection and disconnection events and updates the [`Gamepads`] resource accordingly.
+///
+/// ## Schedule
```

Comment:
Neat idea. I think this sort of thing should be collected centrally in the corresponding plugin though, so we can better manage the bitrot and give users a big-picture view of how the plugin works.